scripts/rabbitmq-service.bat

```diff
@@ -188,6 +188,16 @@ set ERLANG_SERVICE_ARGUMENTS= ^
 set ERLANG_SERVICE_ARGUMENTS=!ERLANG_SERVICE_ARGUMENTS:\=\\!
 set ERLANG_SERVICE_ARGUMENTS=!ERLANG_SERVICE_ARGUMENTS:"=\"!
 
+set ENV_OK=true

```

Comment:
I had  to move this code [here](https://github.com/rabbitmq/rabbitmq-server/blob/rabbitmq-server-659/scripts/rabbitmq-service.bat#L106) (Didn't commit yet). Because [here](https://github.com/rabbitmq/rabbitmq-server/blob/rabbitmq-server-553/scripts/rabbitmq-service.bat#L191) the service is already installed.
On my tests work better, Makes sense for you?